importer/helpers/helpers.go

```diff
@@ -38,16 +38,35 @@ var ErrSizeLimitExceeded = fmt.Errorf("object size limit exceeded")
 // of unixfs DAG trees
 type UnixfsNode struct {
 	node *dag.Node
-	ufmt *ft.MultiBlock
+	ufmt *ft.FSNode
 }
 
 func NewUnixfsNode() *UnixfsNode {
 	return &UnixfsNode{
 		node: new(dag.Node),
-		ufmt: new(ft.MultiBlock),
+		ufmt: &ft.FSNode{Type: ft.TFile},
 	}
 }
 
+func NewUnixfsBlock() *UnixfsNode {
+	return &UnixfsNode{
+		node: new(dag.Node),
+		ufmt: &ft.FSNode{Type: ft.TRaw},
```

Comment:
If i'm reading this correctly... why is there a need for differentiating `ft.TRaw` and `ft.TFile`? 

Making them different types breaks the amazing composability that you get from them being the same. No longer can one file be embedded in another _completely_.